src/app/core/components/Sidenav/WorkspaceSelectorContainer.tsx

```diff
@@ -13,6 +18,22 @@ const WorkspaceSelectorContainer = ({ user }: { user: UserSettings | undefined }
   const parsedWorkspaces = parseWorkspaces(workspaces);
   const parsedPendingWorksapces = parsePendingWorkspaces(pendingWorkspaces);
   const allParsedWorkspaces = [...parsedWorkspaces, ...parsedPendingWorksapces];
+  const [pendingWorkspacesInvites, setPendingWorkspacesInvites] = useState<PendingInvitesResponse>([]);
+  const [isDialogOpen, setIsDialogOpen] = useState<boolean>(false);
+  const [isLoading, setIsLoading] = useState<boolean>(false);
+
+  useEffect(() => {
+    getPendingInvites();
+  }, [isLoading]);
```

Comment:
perhaps it would be better to control this so that it only runs when the selector is opened. Otherwise it might be executed unexpectedly.

Reply:
fixed